src/Dont/Exception/NonSerialisableObject.php

```diff
@@ -8,6 +8,16 @@
 
 class NonSerialisableObject extends LogicException implements ExceptionInterface
 {
+    private const ERROR_TEMPLATE = <<<'ERROR'
```

Comment:
If it's a `public` const, surely you don't need to re-write it out in the test? Or is the reason explicitly to copy it to verify?